integrations/clprf/test_trec_covid_r5.py

```diff
@@ -0,0 +1,140 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
```

Comment:
I think we changed the tagline? https://github.com/castorini/pyserini/blob/master/integrations/clprf/test_trec_covid_r4.py#L2